src/scss/about.scss

```diff
@@ -0,0 +1,87 @@
+@import 'mixins';
+
+.about {
+    &_background {
```

Comment:
Could you explain the advantages of using `&_selector`?

Reply:
The `&_selector` was used in this project to create namespace-like class containers with a lot less repetitive code. For example, if the About section's title and the Features section's title need different styling, they will compile to about_title and features_title by using a parent selector and the `&_selector`.

One of the best uses for it, though, is to avoid repetition in the styling of pseudoclasses:

```css
.button {

  // some style 1

  &:hover {
    // some style 2
  }
  
  &:active { 
    // some style 3
  }
}
```

istead of 

```css
.button {
  // some style 1
}

.button:hover {
 // some style 2
}

.button:active {
 // some style 3
}
```